Src/CSharpier.Tests/FormattingTests/TestFiles/VariableDeclarations.cst

```diff
@@ -76,8 +76,20 @@ class ClassName
                 + SqlParameterCollectionExtensions.DefaultValueColumnWidth
         ];
 
-        var someLongValue_________________ =
-            memberAccessExpression[elementAccessExpression].theMember______________________________;
+        var someLongValue_________________ = memberAccessExpression[
+            elementAccessExpression
+        ].theMember______________________________;
+
+        var someLongValue_________________ = memberAccessExpression[
+            elementAccessExpression
+        ].theMember______________________________();
+
+        var someLongValue_________________ = memberAccessExpression(
+            elementAccessExpression
+        ).theMember______________________________;
+
+        var someLongValue_________________ = memberAccessExpression(elementAccessExpression)
+            .theMember______________________________();
```

Comment:
The old version was better, but was not consistent compared to the rest of these and causing some other weirdness.
The goal of #451 is to get all of these looking more like the last of the 4.